tests/test_fromtable.py

```diff
@@ -172,6 +172,40 @@ def test_wo_singlecolumns():
     assert wateroil.table["PC"].sum() == 0
 
 
+@pytest.mark.parametrize(
+    "h, sw_mid",
+    [
+        (0.1, 0.1),
+        (0.2, 0.1),
+        (0.1, 0.2),
+        (0.1, 0.1000001),
+        (0.1, 0.9999999),
+        (1, 0.1),
+    ],
+)
```

Comment:
There appears to still be some issues with sorw (and swcr), specifically for these tests:
- The sorw value of the created WaterOil object does not (necessarily) equal the result of estimate_sorw() [=1-(swl+h)] ?]
- swcr=1 

In principle I would argue that since sorw=sowcr the value should be defined by the oil curve..

Reply:
estimate_sorw is unreliable on the test input in this test, but a reliable sorw is not needed for add_fromtable(). The intention of the test is to avoid crashes.

wateroil.estimate_sorw() has a docstring outlining how it works, and also mentions why it doesn't use KROW directly.